src/libraries/System.Net.Quic/src/System.Net.Quic.csproj

```diff
@@ -76,11 +78,31 @@
     <Reference Include="System.Threading" />
     <Reference Include="System.Threading.Channels" />
   </ItemGroup>
+
+  <PropertyGroup>
+    <BinPlaceNative>true</BinPlaceNative>
+    <BinPlaceRuntime>false</BinPlaceRuntime>
+  </PropertyGroup>
+
   <!-- Support for deploying msquic -->
+  <ItemGroup Condition="'$(TargetsWindows)' == 'true' and ('$(TargetArchitecture)' == 'x64' or '$(TargetArchitecture)' == 'x86')">
+    <Content Include="$(PkgSystem_Net_MsQuic_Transport)\runtimes\win10-$(TargetArchitecture)\native\msquic.dll" Condition="'$(TargetsWindows)' == 'true'">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+      <TargetPath>MsQuic.Native.dll</TargetPath>
+    </Content>
+    <Content Include="$(PkgSystem_Net_MsQuic_Transport)\runtimes\win10-$(TargetArchitecture)\native\msquic.pdb" Condition="'$(TargetsWindows)' == 'true'">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+      <TargetPath>MsQuic.Native.pdb</TargetPath>
```

Comment:
Is the debugger still able to find the symbols after the rename? IIRC, renaming like this breaks it. Can we build the .dll with the right name in the first place instead?

Reply:
> Can we build the .dll with the right name in the first place instead?

AFAICT no, at least not anyhow easily, but @nibanks will know the definite answer to that.

Reply:
I will take a look. Thanks for the warning @jkotas. We can control the repo producing the NuGet. The priority now is to get CI & stress running so it may be follow-up PR.

Reply:
@ThadHouse would it be difficult to add a cmake flag/option to allow the caller to change `msquic` to `MsQuic.Native` when building the binary?

Reply:
Thats something we could easily do. Probably would have to fix up some scripts, but generally thats easy enough.

Reply:
@wfurt @jkotas Do either of you know how to get a `.` to work in a .def file? I get a linker error if there is a `.` in the library name. If I just rename to something like `MsQuicNative` it works fine.

Reply:
I updated the exclude pattern so we do not need to rename. I think that will be less confusing for everybody.